cli/command/environment_test.go

```diff
@@ -0,0 +1,312 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestEnvironmentCommand_userInputErrors(t *testing.T) {
```

Comment:
From the other prs I’ve seen, looks like the pattern is one “userInputErrors” test per entity command. Also, many of these tests are missing cases (at least one for each missing arg, invalid types, etc.)